### PR TITLE
Update the hero section with the new Nota assets and figma specs

### PR DESCRIPTION
**Pull request by jbirori**
see title

### CODE REVIEW

File: src/components/Button/ButtonStyles.scss [hunk 1]
```diff
@@ -20,8 +20,8 @@
     text-align: center;
     padding: 0;
     margin: 0;
+    margin-bottom: -3px;
```
Comment (jbirori) [inline src/components/Button/ButtonStyles.scss:23]: The subheading font has a slight gap underneath which makes it look uncentered in the button

File: src/images/mobile_hero.png
Comment (bomanimc) [inline src/images/mobile_hero.png:1]: Loooove this

File: src/images/2021_app_lineup.png
Comment (bomanimc) [inline src/images/2021_app_lineup.png:1]: Really minor, but I wonder if an image that doesn't truncate the name of the first artist in the list view would be better here. Probably not very noticeable, but thinking that people may read from top down.

**Review by bomanimc — APPROVED** [review]

File: src/images/2021_app_lineup.png
Comment (jbirori) [reply src/images/2021_app_lineup.png:1]: Good point! I don't seem to have any other screenshots of the 2021 lineup screen on my phone, do you have any you can share? Going to merge as is and we can update in a follow up

**Pull request merged**